airflow/models/trigger.py

```diff
@@ -90,8 +90,17 @@ def __init__(
     @internal_api_call
     def from_object(cls, trigger: BaseTrigger) -> Trigger:
         """Alternative constructor that creates a trigger row based directly off of a Trigger object."""
+        from airflow.models.crypto import get_fernet
+
         classpath, kwargs = trigger.serialize()
-        return cls(classpath=classpath, kwargs=kwargs)
+        secure_kwargs = {}
+        fernet = get_fernet()
+        for k, v in kwargs.items():
+            if k.startswith("encrypted__"):
```

Comment:
The literal `"encrypted__"` is used now across the code multiple times, can you make this a constant and all parts of the code refer to the same constant?

Reply:
Good idea, just updated it.